application.py

```diff
@@ -28,11 +30,25 @@ def index():
             # Use this when resubmitting code to run
             key = r.get('key')
 
-            return render_template('base.html', template='code.html', fixed=fixed, result=result, error=error)
+            return render_template('base.html',
+                                   template='code.html',
+                                   fixed=fixed,
+                                   result=result,
+                                   error=error,
+                                   key=key)
     else:
         return render_template('base.html', template='index.html')
 
 
+@app.route('/resubmit', methods=['POST'])
+def resubmit():
+    if request.method == 'POST':
+        r = requests.post('http://127.0.0.1:5000/api/resubmit_code',
```

Comment:
Is this the correct endpoint here?

Reply:
Nope, my mistake will be changing that after the MR on WLC is done so I can test there 🙂

Reply:
Meged @SuNaden